scripts/change_address_types.py

```diff
@@ -0,0 +1,47 @@
+# encoding=utf-8
+
+from sefaria.helper.schema import *
```

Comment:
In the future, it's better to explicitly say which functions you need because they can potentially overwrite variables in your script when you import with `*`